services/src/handlers/wfs.rs

```diff
@@ -416,8 +436,8 @@ async fn get_feature<C: Context>(
 
     // TODO: validate request further
 
-    if request.type_names.feature_type == "93d6785e-5eea-4e0e-8074-e7f78733d988" {
-        return get_feature_mock(request);
+    if request.typeNames.feature_type == "93d6785e-5eea-4e0e-8074-e7f78733d988" {
```

Comment:
Please use snake_case

Reply:
The OGC API specification wants it to be camelcase and we cannot rewrite it. cf. my TODO in the ogc/wfs/request.rs